nutrinova-ui/src/services/customerService.ts

```diff
@@ -8,25 +8,15 @@ export interface Customer {
 const userService = {
   async customerExistsServer(id: string): Promise<boolean> {
     'use server'
-    try {
```

Comment:
Did you get rid of the try and catch because it should be handled on the api?

Reply:
Catching an error and throwing a new one was not very helpful because it swallows the stack trace and error message. It makes it easier to see why it failed in the error boundary while were developing, and it will get hidden in prod anyway. I realized that it really made no sense that I tried to catch it here because it's not like the app can continue without it anyway.